pkg/bbgo/position.go

```diff
@@ -33,6 +33,22 @@ type Position struct {
 
 	ExchangeFeeRates map[types.ExchangeName]ExchangeFee `json:"exchangeFeeRates"`
 
+	// Futures data fields
+	Isolated               bool             		  `json:"isolated"`
+	Leverage               fixedpoint.Value           `json:"leverage"`
+	InitialMargin          fixedpoint.Value           `json:"initialMargin"`
+	MaintMargin            fixedpoint.Value           `json:"maintMargin"`
+	OpenOrderInitialMargin fixedpoint.Value           `json:"openOrderInitialMargin"`
+	PositionInitialMargin  fixedpoint.Value           `json:"positionInitialMargin"`
+	UnrealizedProfit       fixedpoint.Value           `json:"unrealizedProfit"`
+	EntryPrice             fixedpoint.Value           `json:"entryPrice"`
+	MaxNotional            fixedpoint.Value           `json:"maxNotional"`
+	PositionSide           string 					  `json:"positionSide"`
+	PositionAmt            fixedpoint.Value           `json:"positionAmt"`
+	Notional               fixedpoint.Value           `json:"notional"`
+	IsolatedWallet         fixedpoint.Value           `json:"isolatedWallet"`
+	UpdateTime             int64           			   `json:"updateTime"`
```

Comment:
I think these fields are binance specific, could you please move the position type to the binance exchange package? we can convert it to the global type later.

Reply:
OK. Maybe I can just remove them.
Binance futures related fields of Position in use is now cloned in types package.
I can move them to local type later. As you mentioned, convert it to the global type then.

Reply:
Done